Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -236,8 +246,53 @@ props.buildReportOrder.each { buildReportFile ->
 				}
 			}
 		}
+
+		deletionRecords.each { deleteRecord ->
+				deletionCount += deleteRecord.getAttributeAsList("deletedBuildOutputs").size()
+        		deleteRecord.getAttributeAsList("deletedBuildOutputs").each{ deletedFile ->
+            		String cleansedDeletedFile = ((String) deletedFile).replace('"', '');
+					def (dataset, member) = getDatasetName(cleansedDeletedFile)
+					
+					// search for an existing deployableArtifacts record
+				ArrayList<DeployableArtifact> filteredDeployableArtifacts = new ArrayList()
+				buildOutputsMap.each { DeployableArtifact dA, info ->
```

Comment:
Can you switch to a more complete variable name instead of ```dA```? The next line use ```deployableArtifact```, for readibility.